### ISSUE TITLE
xselect not working on simulated data

### ISSUE THREAD

**Issue by lucabaldini** [lucabaldini/ixpeobssim#588]
```
Command not found; type ? for a command listing
xsel24181:IXPE-DU1 > read eve
> Enter Event file list >[toy_point_source_du1_nomc.fits]

 Notes: XSELECT set up for      IXPE
 Time keyword is TIME       in units of s
 Default timing binsize =   5.0000

Setting...
 Image  keywords   = X          Y           with binning =    1
 WMAP   keywords   = X          Y           with binning =    1
 Energy keyword   = PI                     with binning =    1

Getting Min and Max for Energy Column...

Could not find size keywords for column PI for file:
/Users/doroshv/Documents/Projects/ixpe/unbinned_analysis/toy_point_source_du1_nomc.fits
Tried TLMIN, TLMAX and TLMAX
I will remove this file, as it will cause problems at the extract stage.
No files added
xsel24181:IXPE-DU1 >
xsel24181:IXPE-DU1 > extr ima
No data has been read in yet
 XSL_BIN: Status =    -5
xsel24181:IXPE-DU1 > pl ima 
```

This was reported by Victor at the SASWG telecon on March 31, 2022.

Original url: https://bitbucket.org/ixpesw/ixpeobssim/issues/588
Created on 2022-04-01T05:02:40.259282+00:00

**Comment by lucabaldini** [comment]
Ok, I just verified that this is fixed after https://github.com/lucabaldini/ixpeobssim/pull/689

**Comment by lucabaldini** [comment]
Shipped in version 30.2.0
